src/docs/asciidoc/core/core-expressions.adoc

```diff
@@ -1820,8 +1820,8 @@ selection lets us easily get a list of Serbian inventors, as the following examp
 			"members.?[nationality == 'Serbian']").getValue(societyContext) as List<Inventor>
 ----
 
-Selection is possible upon both lists and maps. For a list, the selection
-criteria is evaluated against each individual list element. Against a map, the
+Selection is possible upon lists, arrays and maps. For a list or array, the selection
```

Comment:
Selection is actually supported for arrays and anything that implements `java.lang.Iterable` or `java.util.Map`.

Would you like to update your PR to reflect that?

Reply:
Done.